bots/engine.py

```diff
@@ -85,17 +85,19 @@ def start():
         botsglobal.logger = botsinit.initenginelogging(process_name)
         atexit.register(logging.shutdown)
     except Exception as msg:
+        print('Error initialising logging:' + msg)
         botslib.sendbotserrorreport('[Bots severe error] Bots is not running anymore','Bots does not run because logging is not possible.\nOften a rights problem.\n')
         sys.exit(1)
     else:
-        if botsglobal.ini.get('settings','log_file_number','') != 'daily':
+        if botsglobal.ini.get('settings','log_when',None) != 'daily':
             for key,value in botslib.botsinfo():    #log info about environement, versions, etc
                 botsglobal.logger.info('%(key)s: "%(value)s".',{'key':key,'value':value})
 
     #**************connect to database**********************************
     try:
         botsinit.connect()
     except Exception as msg:
+        print('Error connecting to database:' + msg)
```

Comment:
not needed, is via logger

Reply:
Added these for quicker debugging while testing, it's good to see errors on the console